src/main/java/com/example/user_management/api/exception/UserUnauthorizedException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.user_management.api.exception;
+
+public class UserUnauthorizedException extends RuntimeException {
```

Comment:
It will be great to have the userId as a prop for this exception and dedicated message that will include info about userId or which user was not found.